Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.py

```diff
@@ -1221,48 +1444,340 @@ def search_records_command(args: dict[str, Any]) -> CommandResults:
+        addComponents=argToList(args.get("add_components")),
+        deleteComponents=argToList(args.get("delete_components")),
+        ttl=args.get("ttl"),
+    )
+    if args.get("context_output", "false") == "true":
+        return CommandResults(
+            readable_output=pretty_print_asset([result["data"]]),
+            outputs_prefix="Argus.Asset",
+            outputs=result,
+            raw_response=result,
+        )
+    return CommandResults(readable_output=pretty_print_asset([result["data"]]))
+
+
+def get_vulnerability_by_asset_command(args: dict) -> CommandResults:
+    id_shortname = args.get("id_shortname")
+    context_output = args.get("context_output", "false")
+
+    if not id_shortname:
+        raise ValueError("Asset id or shortname not given")
+
+    result = get_vulnerability_by_asset(
+        asset=id_shortname,
+        customer=args.get("customer"),
+        includeDeleted=args.get("include_deleted"),
+        offset=args.get("offset"),
+        limit=args.get("limit"),
+    )
+    if context_output == "true":
+        return CommandResults(
+            readable_output=pretty_print_vulnerability([result["data"]]),
+            outputs_prefix="Argus.Asset",
+            outputs=result,
+            raw_response=result,
+        )
+    return CommandResults(readable_output=pretty_print_vulnerability(result["data"]))
+
+
+def get_vulnerability_by_id_command(args: dict) -> CommandResults:
+    vulnerability_uuid = args.get("vulnerability_uuid")
+    context_output = args.get("context_output", "false")
+
+    if not vulnerability_uuid:
+        raise ValueError("Vulnerability UUID not given")
+
+    result = get_vulnerability_by_id(id=vulnerability_uuid)
+    if context_output == "true":
+        return CommandResults(
+            readable_output=pretty_print_vulnerability([result["data"]]),
+            outputs_prefix="Argus.Vulnerability",
+            outputs=result,
+            raw_response=result,
+        )
+    return CommandResults(readable_output=pretty_print_vulnerability([result["data"]]))
+
+
+def list_vulnerabilities_command(args: dict) -> CommandResults:
+    context_output = args.get("context_output", "false")
+
+    result = list_vulnerabilities(
+        asset=args.get("asset"),
+        customer=args.get("customer"),
+        vulnerabilityID=args.get("vulnerability_id"),
+        severity=args.get("severity"),
+        resolution=args.get("resolution"),
+        limit=args.get("limit"),
+        offset=args.get("offset"),
+        includeDeleted=args.get("include_deleted"),
+        startTimestamp=args.get("start_timestamp"),
+        endTimestamp=args.get("end_timestamp"),
+        timeFieldStrategy=args.get("time_field_strategy"),
+        sortBy=args.get("sort_by"),
     )
+    if context_output == "true":
+        return CommandResults(
+            readable_output=pretty_print_vulnerability([result["data"]]),
+            outputs_prefix="Argus.Asset",
```

Comment:
The context path prefix defined in the yml for that command are `Argus.Vulnerability`